README.md

```diff
@@ -30,48 +30,93 @@ Options:
 
 ### Modes
 
-Edge supports various modes, from a local [offline](#offline) mode to a full blown [edge](#edge) mode supporting dynamic keys, metrics and other advanced features.
+Edge currently supports 2 different modes:
 
-#### Offline
+- [Edge](#edge) - Connection to upstream node (Unleash instance or another Edge). Supports dynamic tokens, metrics and other advanced features;
+- [Offline](#offline) - No connection to upstream node. Full control of data and tokens;
+
+#### Edge
+
+```mermaid
+graph LR
+  A(Client) -->|Fetch toggles| B((Edge))
+  B-->|Fetch features| C((Unleash))
+```
 
-You have a need to have full control of both the data your clients will get and which keys can be used to access the server. This mode needs a downloaded JSON dump of a result from a query against an Unleash server on the [/api/client/features](https://docs.getunleash.io/reference/api/unleash/get-client-feature) endpoint as well as a comma-separated list of keys that should be allowed to access the server.
+Edge mode is the "standard" mode for Unleash Edge and the one you should default to in most cases. It connects to an upstream node, such as your Unleash instance, and uses that as the source of truth for feature toggles.
 
-If your keys follow the Unleash API key format `[project]:[environment].<somesecret>`, Edge will filter the features dump to match the project contained in the key.
+Another than connecting Edge directly to your Unleash instance, it's also possible to connect to another Edge instance (_daisy chaining_). You can have as many Edge nodes as you'd like between the Edge node your clients are accessing and the Unleash server, and it's also possible for multiple nodes to connect to a single upstream one. Depending on your architecture and requirements this can be a powerful feature, offering you flexibility and scalability when planning your implementation.
```

Comment:
```suggestion
Other than connecting Edge directly to your Unleash instance, it's also possible to connect to another Edge instance (_daisy chaining_). You can have as many Edge nodes as you'd like between the Edge node your clients are accessing and the Unleash server, and it's also possible for multiple nodes to connect to a single upstream one. Depending on your architecture and requirements this can be a powerful feature, offering you flexibility and scalability when planning your implementation.
```

Reply:
Nice catch!